packages/@aws-cdk/aws-ecs/lib/ec2/ec2-service.ts

```diff
@@ -70,6 +70,14 @@ export class Ec2Service extends BaseService implements elb.ILoadBalancerTarget {
       throw new Error('Daemon mode launches one task on every instance. Don\'t supply desiredCount.');
     }
 
+    if (props.daemon && props.maximumPercent !== undefined && props.maximumPercent !== 100) {
```

Comment:
Document these defaults/constraints in Props

Reply:
done!

Reply:
I would also document the requirement that this has to be 100/0 for daemon mode (not just that it’s the default)